lib/coverage-instrumenter.js

```diff
@@ -24,25 +25,35 @@ function getPathForRealFile(relativePath, root, templateExtensions) {
   return null;
 }
 
-function fixPath(relativePath, name, root, templateExtensions) {
+function fixPath(relativePath, name, root, templateExtensions, isAddon) {
   // Handle apps (served from app name)
-  if (relativePath.indexOf(name) === 0) {
-    relativePath = relativePath.replace(name, 'app')
-    return getPathForRealFile(relativePath, root, templateExtensions) || relativePath
+  if (!isAddon && relativePath.includes(name)) {
+    relativePath = relativePath.replace(name, 'app');
+    return getPathForRealFile(relativePath, root, templateExtensions) || relativePath;
   }
 
   // Handle addons (served from dummy app)
-  if (relativePath.indexOf('dummy') === 0) {
-    relativePath = relativePath.replace('dummy', 'app')
-    var dummyPath = path.join('tests', 'dummy', relativePath)
+  if (relativePath.includes('dummy')) {
```

Comment:
Like above, it should be `indexOf(name) === 0` or `.startsWith(name)`.  Also, should change the conditional to only do this when `isAddon`.